index.html

```diff
@@ -72,7 +72,7 @@ <h3>the lightweight<br>YouTube experience<br>for Android</h3>
             <div id="text-1-wrapper">
                 <div class="row">
                     <div class="col-sm-6">
-                        <p><strong>Do you like watching videos on YouTube<br>but are looking for a simple app to do that?</strong></p>
+                        <p><strong>Do you like watching videos on YouTube<br>but look for a simpler app to do that?</strong></p>
```

Comment:
I think we should just put a `and` for that `but`, then the sentence makes more sense in general.
I'm not sure about changing present participle for simple present. What's the rationale behind this change?
I don't think we should put any sort of "weighting" into this sentence. Who says NewPipe is simpler than app XYZ? We need to be careful about this.

Reply:
I can imagine the but comes from using their website/app.
Oh grr, now I have to argue in english grammar. I think it is a participle phrase I want here? The watching part describes the app more closely, which means we don't need the participle in the main phrase. Maybe replace look with want, that could help my point.

Tbh, lets rephrase that whole second sentence, I think simple is in general not appropriate.
Do you like watching videos on YouTube _but want an inutitive, feature-rich app for that?_

Reply:
We might add `and privacy friendly` to your suggestion, then it's perfect.

Reply:
@TheAssassin That sounds good

Reply:
Roger that, will put this in my next commit.

I will mark this as resolved so its out of the overview, if thats okay. Otherwise scream and reopen it ;P